tests/app/dao/test_events_dao.py

```diff
@@ -1,16 +1,22 @@
+from sqlalchemy import func, select
+
 from app.dao.events_dao import dao_create_event
 from app.models import Event
 
 
 def test_create_event(notify_db_session):
-    assert Event.query.count() == 0
+    stmt = select(func.count()).select_from(Event)
```

Comment:
Is this the only `Event` test? This will fail if there are any other tests that use the `Event` table.

Reply:
Could preemptively mark it serial?

Reply:
It is the only test in that module and, yes, it could  fail in parallel.  I have only updated syntax.  Changing logic would be an expansion of scope.  If that is desirable, the ticket should be repointed to a 5.

Reply:
Shall I mark it as serial?

Reply:
I will mark it as serial.